proofs/eo/cpc/theories/Strings.eo

```diff
@@ -57,7 +74,7 @@
 ;   declare it to be :right-assoc-nil to model cvc5's treatment of variadic
 ;   functions.
 (declare-parameterized-const str.++ ((T Type))
```

Comment:
Should this be renamed?

Reply:
str.++ and seq.++ are the same operator.

At some point I will refactor the entire signature so that the former is defined as the latter, and not vice versa. Will do this in a future PR.